tracing_test.go

```diff
@@ -546,12 +542,16 @@ func TestContinueTransactionFromHeaders(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		s := &Span{isTransaction: true}
+		s := &Span{}
+		s.recorder = &spanRecorder{}
```

Comment:
This does make testing more awkward. Could probably have a test helper to create a span that's a transaction.